client/allocrunner/networking_bridge_linux.go

```diff
@@ -207,6 +243,11 @@ const nomadCNIConfigTemplate = `{
 			"backend": "iptables",
 			"iptablesAdminChainName": %q
 		},
+		{
+			"type": "firewall",
+			"backend": "ip6tables",
```

Comment:
I remember adding this, but at the time I think we were kinda flinging spaghetti at the wall to see what stuck.  is this actually needed?  and again, is it ok on systems without ipv6?

Reply:
Actually, just checked, there's no need for it! Thanks for the heads up :)